### PR TITLE
Reorder tutorials in packages and add readme

### PR DESCRIPTION
**Pull request by reuvenperetz**
## Pull Request Description:
Reorder tutorials in packages and add readme.

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+Dive into MCT with our collection of tutorials, covering a wide 
+range of compression techniques for Keras and Pytorch models. We provide
+both Python scripts and interactive Jupyter notebooks for an
+engaging and hands-on experience.
+
+
+## Keras Tutorials
+
+<details id="keras-ptq">
+  <summary>Post-Training Quantization</summary>
```
Comment (lior-dikstein) [inline tutorials/notebooks/README.md:29]: Post-Training Quantization (PTQ). Do this for QAT and all acronyms you use in the features section.

File: tutorials/notebooks/README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  - [Debug Tools](#keras-debug)
+- [Pytorch Tutorials](#pytorch-tutorials)
+  - [Quick-Start with Torchvision](#pytorch-quickstart-torchvision)
+  - [Post-Training Quantization](#pytorch-ptq)
+  - [Quantization-Aware Training](#pytorch-qat)
+  - [Data Generation](#pytorch-data-generation)
+  - [Export Quantized Models](#pytorch-export)
+
+## Introduction
+Dive into MCT with our collection of tutorials, covering a wide
```
Comment (lior-dikstein) [inline tutorials/notebooks/README.md:20]: into the Model Compression Toolkit (MCT)

File: tutorials/notebooks/README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+</details>
+
+<details id="pytorch-data-generation">
+  <summary>Data Generation</summary>
+  
+  | Tutorial                                                                                                                          | Included Features                                              |
+  |-----------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------|
+  | [Data-Free Quantization using Data Generation](tutorials/notebooks/pytorch/data_generation/example_pytorch_data_generation.ipynb) | &#x2705; Data-Free Quantization <br/> &#x2705; Data Generation |
+
```
Comment (lior-dikstein) [inline tutorials/notebooks/README.md:128]: Features: Data Generation, PTQ

File: tutorials/notebooks/README.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+## Pytorch Tutorials
+
+
+<details id="pytorch-quickstart-torchvision">
+  <summary>Quick-Start with Torchvision</summary>
+  
+  | Tutorial                                                                                                        |
+  |-----------------------------------------------------------------------------------------------------------------|
+  | [Quick Start - Torchvision Pretrained Model](tutorials/notebooks/pytorch/example_quick_start_torchvision.ipynb) |
+
```
Comment (lior-dikstein) [inline tutorials/notebooks/README.md:96]: No features?

**Review by lior-dikstein — APPROVED** [review]

File: tutorials/notebooks/README.md (hunk 4 above)
Comment (reuvenperetz) [reply tutorials/notebooks/README.md:96]: No, it does not demonstrate any specific feature.

**Pull request merged**